csharp/ql/lib/semmle/code/csharp/security/dataflow/flowsources/Stored.qll

```diff
@@ -60,7 +60,7 @@ class DbDataReaderPropertyStoredFlowSource extends DatabaseInputSource {
 }
 
 /** A read of a mapped property. */
-class ORMMappedProperty extends DatabaseInputSource {
+deprecated class ORMMappedProperty extends DataFlow::Node {
```

Comment:
Maybe consider to add a deprecation message in the comments (use `EntityFrameWork::StoredFlowsource` and `NHibernate::StoredFlowSource` instead).

Reply:
Done.